scripts/build_posix.sh

```diff
@@ -101,6 +102,11 @@ install_dependencies()
 build_dvc()
 {
 	print_info "Building dvc binary..."
+
+  if [[ "$(uname)" != 'Linux' ]]; then
```

Comment:
Offset is off :) You need to use `tab` here instead of spaces to match the style around it.